### PR TITLE
Add Immich Kiosk

### PR DESCRIPTION
**Pull request by damongolding**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/immich-kiosk.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Immich Kiosk
+website_url: https://github.com/damongolding/immich-kiosk
+source_code_url: https://github.com/damongolding/immich-kiosk
+description: Immich Kiosk is a lightweight slideshow for running on kiosk devices and browsers that uses Immich as a data source.
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+  - GO
```
Comment (kokomo123) [inline software/immich-kiosk.yml:9]:
```suggestion
  - Go
```
Just a small grammatical adjustment.

File: software/immich-kiosk.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: Immich Kiosk
+website_url: https://github.com/damongolding/immich-kiosk
+source_code_url: https://github.com/damongolding/immich-kiosk
+description: Immich Kiosk is a lightweight slideshow for running on kiosk devices and browsers that uses Immich as a data source.
```
Comment (Rabenherz112) [inline software/immich-kiosk.yml:4]:
```suggestion
description: A lightweight slideshow for running on kiosk devices and browsers that uses Immich as a data source.
```

File: software/immich-kiosk.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+licenses:
+  - AGPL-3.0
```
Comment (Rabenherz112) [inline software/immich-kiosk.yml:6]:
```suggestion
  - GPL-3.0
```

**Review by Rabenherz112 — COMMENTED** [review]
Some smaller changes

File: software/immich-kiosk.yml (hunk 3 above)
Comment (damongolding) [reply software/immich-kiosk.yml:6]: I can't believe I got my own license wrong 🤦

File: software/immich-kiosk.yml (hunk 2 above)
Comment (damongolding) [reply software/immich-kiosk.yml]: Reads much better. Thank you.

**Review by Rabenherz112 — APPROVED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**